.github/workflows/trunk.yml

```diff
@@ -320,4 +320,32 @@ jobs:
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-linux.sh cmake
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-qnn-deps.sh
         PYTHON_EXECUTABLE=python bash .ci/scripts/build-qnn-sdk.sh
-        PYTHON_EXECUTABLE=python bash .ci/scripts/test.sh ${{ matrix.model }} "cmake" "qnn"
+        PYTHON_EXECUTABLE=python bash .ci/scripts/test_model.sh ${{ matrix.model }} "cmake" "qnn"
+
+  test-coreml-model:
+    name: test-coreml-model
+    uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
+    strategy:
+      matrix:
+        model: [mv3, ic4, resnet50, edsr, mobilebert, w2l]
```

Comment:
As macos runner is not plentiful, let's not use matrix strategy here and end up with many small jobs.  Instead, you can have one single big macos job that loop through all the models and export them.  That would be more efficient.